docs/pages/application-access/getting-started.mdx

```diff
@@ -40,8 +40,8 @@ expose, skip over to **Step 2**.
 Grafana can be launched in a [Docker container](https://grafana.com/docs/grafana/latest/installation/docker/)
 with a single command:
 
-```sh
-$ docker run -d -p 3000:3000 grafana/grafana
+```bash
+docker run -d -p 127.0.0.1:3000:3000 grafana/grafana
 ```
```

Comment:
The video indicates that this should be done using a cloud VM. I think to get an SSL through ACME the host must be something other than localhost. So, totally agree and think we need to mention this better.

The command above will prevent Grafana from displaying if the port mapping isn't `127.0.0.1:3000`.